src/Strings.ts

```diff
@@ -307,15 +322,13 @@ export const strings = (locale: string) => {
         locale = 'el-gr';
     else if (locale.startsWith('it'))
         locale = 'it-it';
-    else if (locale === 'zh-yue')
-        locale = 'zh-yue';
     else if (locale === 'zh-hant' || locale === 'zh-hk' || locale === 'zh-mo' || locale === 'zh-tw')
         locale = 'zh-hant';
     else if (locale.startsWith('zh'))
         locale = 'zh-hans';
     else if (locale.startsWith('cs'))
         locale = 'cs-cz';
-    else
+    else if (locale in localizedStrings === false)
         locale = 'en-us';
```

Comment:
If locale is not defined, It will be `en-us` as default.
Please let me know if it's not expected behavior.

Reply:
This is a better fix, thank you.